packages/snaps-sdk/src/types/methods/revoke-permissions.ts

```diff
@@ -0,0 +1,19 @@
+import type { InitialPermissions } from '../permissions';
+
+/**
+ * The request parameters for the `snap_revokePermissions` method.
+ *
+ */
+export type RevokePermissionsParams = InitialPermissions;
+
+/**
+ * The revokePermissions hook parameters used by the `snap_revokePermissions` method.
+ *
+ */
+export type RevokePermissionsHookParams = (keyof InitialPermissions)[];
```

Comment:
Do we need this in the SDK or could it be defined with the RPC method instead?

Reply:
Moved to RPC method file.